controllers/alphacontrollers/alert_controller.go

```diff
@@ -992,8 +995,17 @@ func flattenNotification(notification *alerts.AlertNotification) coralogixv1alph
 
 	switch integration := notification.GetIntegrationType().(type) {
 	case *alerts.AlertNotification_IntegrationId:
```

Comment:
Should this still exist, even if we're removing the integration id?

Reply:
It suppose to be nil when you remove it